girdermedviewer/app/vtk/utils.py

```diff
@@ -598,17 +600,19 @@ def load_volume(file_path):
                 image_data, _ = exporter.readDICOMVolume(folder)
                 return image_data
 
+    if file_path.endswith(".vti"):
+        reader = vtkXMLImageDataReader()
+        reader.SetFileName(file_path)
+        reader.Update()
+        return reader.GetOutput()
+
     raise Exception("File format is not handled for {}".format(file_path))
 
 
 def load_mesh(file_path):
     """Read a file and return a vtkPolyData object"""
     logger.info(f"Loading mesh {file_path}")
-    if file_path.endswith(".stl"):
-        reader = vtkSTLReader()
-        reader.SetFileName(file_path)
-        reader.Update()
-
+    def set_transform(reader):
```

Comment:
`set_transform` -> `invert_xy`

But I'm not sure it should be called for vtp file